test/common/deployer.go

```diff
@@ -58,25 +55,22 @@ func DeployDPoSSGNContracts(sgnParams *SGNParams) (mainchain.Addr, mainchain.Add
 		sgnParams.IncreaseRateWaitTime,
 		sgnParams.SidechainGoLiveTimeout)
 	ChkErr(err, "failed to deploy DPoS contract")
-	WaitMinedWithChk(ctx, EtherBase.Client, tx, 0, "Deploy DPoS "+dposAddr.Hex())
 
-	ctx = context.Background()
 	sgnAddr, tx, _, err := mainchain.DeploySGN(EtherBase.Auth, EtherBase.Client, sgnParams.CelrAddr, dposAddr)
 	ChkErr(err, "failed to deploy SGN contract")
-	WaitMinedWithChk(ctx, EtherBase.Client, tx, 0, "Deploy SGN "+sgnAddr.Hex())
 
 	// TODO: register SGN address on DPoS contract
-	ctx = context.Background()
 	dpos, err := mainchain.NewDPoS(dposAddr, EtherBase.Client)
 	ChkErr(err, "failed to new DPoS instance")
+	EtherBase.Auth.GasLimit = 8000000
 	tx, err = dpos.RegisterSidechain(EtherBase.Auth, sgnAddr)
+	EtherBase.Auth.GasLimit = 0
```

Comment:
What is the reason for this change?

Reply:
To disable `EstimateGas`. Otherwise the call will fail locally.